ddtrace/internal/sampling.py

```diff
@@ -180,3 +189,67 @@ def apply_span_sampling_tags(self, span):
         span.set_metric(_SINGLE_SPAN_SAMPLING_RATE, self._sample_rate)
         if self._max_per_second:
             span.set_metric(_SINGLE_SPAN_SAMPLING_MAX_PER_SEC, self._max_per_second)
+
+
+class SpanSamplingRules:
```

Comment:
Why have this as a class?

Based on how it is used, an instance of this class isn't really useful, it seems we only really need the behavior provided by `_make_rules`?

A few options:

- Keep this as a class for organizational reason, but move the methods to be static methods, so you so `SpanSamplingRules.get_single_span_sampling_rules()` instead
- Make it be a method level function instead of introducing the class
- Make a sub-module for span sampling to isolate the helper functions there